contracts/VotingContract.sol

```diff
@@ -37,8 +37,8 @@ contract VotingContract {
         bool _isPrivate,
         address[] _permissions) public {
 
-        require(_votingEndTime > now, "Voting end time has to be somwhere in the future");
-        require(_votingEndTime < _resultsEndTime, "Voting end time has to be earlier than results end time");
+        require(_votingEndTime >= now + 20, "Voting end time has to be somewhere in the future (at least 20s from now)");
```

Comment:
20s is probably fine, but kinda hardcore. Vote with voting time = 20sec does not sound realistic :dagger: . Therefore we should remember about it while preparing front-end logic.

Reply:
First of all, there was no minimum requirement before, so your argument is not vaild -- we weren't going to create 0s votings. This requirement was added when I realized that we should make sure **at least one block will be mined** before `votingEndTime`.
So because a new block is mined roughly every 15s, 20s is a minimum requirement.
Secondly, there is no use in lengthening this period, because if somebody wants to have longer votings, they can already do it.
Thirdly, if we make this period longer, testing VotingContracts will take more time.